### ISSUE TITLE
Error: This response does not support redirection. 

### ISSUE THREAD

**Issue by Mark-H** [thephpleague/omnipay-mollie#13]
Getting the following exception when trying to do a purchase call: `Omnipay\Common\Exception\RuntimeException: Error: This response does not support redirection. `

when using something like this (per the omnipay example):
```php
if ($response->isRedirect()) {
    // redirect to offsite payment gateway
    $response->redirect();
} else {
    // payment failed: display message to customer
    echo $response->getMessage();
}
```

The exception is coming from https://github.com/thephpleague/omnipay-common/blob/master/src/Omnipay/Common/Message/AbstractResponse.php#L75 where apparently we're not implementing the right interface (as `isRedirect()` is return true).

**Comment by eileenmcnaughton** [comment]
I just hit this too - line number above has changed slightly - but not passing this test

$this instanceof RedirectResponseInterface

**Comment by eileenmcnaughton** [comment]
looks like I found an easy fix - PR coming

**Comment by eileenmcnaughton** [comment]
https://github.com/thephpleague/omnipay-mollie/pull/14

Note that it is possible the extends statement should go on the PurchaseResponse ...

**Comment by frederikbosch** [comment]
@kayladnls I believe this can be closed due to the merge of #14.
